admin/projects.go

```diff
@@ -118,6 +119,208 @@ func (s *Service) TeardownProject(ctx context.Context, p *database.Project) erro
 	return nil
 }
 
+// New method for trigger redeploy (tear down instance and deploy again)
+func (s *Service) TriggerRedeploy(ctx context.Context, orgName, name string) error {
+	// Run it all in the background
+	go func() {
```

Comment:
This seems to just copy paste code for tearing down and creating a deployment. Why not call those two functions instead of copy-pasting?

Reply:
Those two functions are deleting and creating the project, which we don't want right? we just want to tear down the instance and provision it again. so i created a new method.